python/ray/_private/services.py

```diff
@@ -1676,7 +1676,7 @@ def start_raylet(
     )
 
     if storage is not None:
-        start_worker_command.append(f"--storage={storage}")
+        start_worker_command.append(f'--storage="{storage}"')
```

Comment:
I think if you do `f"--storage='{stroage}'"`, you may not need the `strip` change.